.postgraphilerc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     // disableQueryLog: true,
     graphileBuildOptions: {
       connectionFilterRelations: true,
+      connectionFilterAllowEmptyObjectInput: true,
```

Comment:
This makes life a whole lot easier. There was an ugly bit of logic in the front end that put a dummy filter in if it was empty, but it's much nicer just to allow empty filter objects here.